### ISSUE TITLE
New window option for "Open all on click"

### ISSUE THREAD

**Issue by archiif** [Pitmairen/selection-search#110]
It'd be very convenient to have a "New window" child option under the submenu's "Open all on click" option, to force all the search engines inside that submenu to be opened in a new window.

**Comment by Pitmairen** [comment]
Yes, I think this could be useful. It shouldn't be too hard to add I think.
